config/config.yml

```diff
@@ -15,17 +15,20 @@ ELASTICSEARCH:
     INDEX: dane-index-k8s
 FILE_SYSTEM:
     BASE_MOUNT: data # data when running S3 integration test, /data when running via Docker
+    BASE_MOUNT_MODEL: model
     INPUT_DIR: input-files
     OUTPUT_DIR: output-files
 INPUT:
     TEST_INPUT_PATH: testsource__testcarrier/inputfile.wav
+    TEST_MODEL_PATH: s3_model/
+    MODEL_S3_URI: s3://model/model.tar.gz
```

Comment:
It feels a bit tricky that you can specify both a model URI here, and a model version in the WHISPER_ASR_SETTINGS. 
Maybe it would be better to specify the S3 bucket where the models are stored, and use the model version names for the model files so that the WHISPER_ASR_SETTINGS.MODEL_VERSION still has effect when using models stored in S3

Reply:
So this line should be removed here, right?